packages/debian/download_packages.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# coding: utf-8
+
+"""Script that download a list of packages, and all of their dependencies."""
+
+import os
+import sys
+import subprocess
+from typing import Mapping, Sequence
+import urllib.parse
+import urllib.request
+
+
+# Helpers {{{
+
+
+def apt_key_add(key_url: str) -> None:
+    """Add the GPG key at `key_url` to the list of trusted keys."""
+    with urllib.request.urlopen(key_url) as response:
+        key = response.read()
+    subprocess.run(['apt-key', 'add', '-'], input=key, check=True)
+
+
+def add_source_list(name: str, url: str) -> None:
+    """Add a new packages source called `name` using the give URL as source."""
+    filepath = '/etc/apt/sources.list.d/{}.list'.format(name)
+    with open(filepath, 'w', encoding='utf-8') as fp:
+        print(url, file=fp)
+
+
+# }}}
+
+
+def add_external_repositories(salt_version: str) -> None:
```

Comment:
We may want to declare these externally at some point

Reply:
Yeah, `version.py` may be a good place.
I'll add a comment.